src/game_api/rpc.cpp

```diff
@@ -806,7 +806,11 @@ void modify_sparktraps(float angle_increment, float distance)
     static float* new_array;
     if (new_array == nullptr)
     {
-        const auto offset = get_address("sparktrap_angle_increment");
+        const auto offset = get_address("sparktrap_angle_increment") + 4;

```

Comment:
Make this static?

Reply:
aren't addresses in get_address cached?

Reply:
I guess, but they still need to be looked up in a map. Idk what's more efficient, static vars or map lookups.

Reply:
Static should be faster, not that it matters in such non-frequent used function